src/main/scala/components/control.scala

```diff
@@ -49,7 +49,7 @@ class Control extends Module {
 
   val signals =
     ListLookup(io.opcode,
-      /*default*/           List(false.B, false.B,   false.B, false.B,   false.B,  false.B,  0.U,   false.B,      false.B, false.B, false.B, 0.U,   false.B),
+      /*default*/           List(false.B, false.B,   false.B, false.B,   false.B,  false.B,  0.U,   0.B,      false.B, false.B, false.B, 0.U,   false.B),
```

Comment:
Nice catch! Could you also update it to .U for the integer type?

Reply:
Done!